package.json

```
@@ -68,5 +68,9 @@
     "tsdx": "^0.14.1",
     "tslib": "^2.3.1",
     "typescript": "^3.9.10"
+  },
+  "peerDependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
```

Peers describe the version that users need to bring, so it's nice to make them as wide as possible. I don't know if this actually works with v16, but if it does we might do something like this:
```suggestion
    "react": "^16.0.0 || ^17.0.0",
    "react-dom": "^16.0.0 || ^17.0.0"
```

If we don't wanna claim that this works for 16 then `^17.0.0` is probably what you want unless there's something interesting about `>=17.0.1` that this lib depends on.

---

We use hooks in our React component for embed so I tested this out and it works for 16.8 +.

---

Perfect!